test/test-helpers/fake-data.js

```diff
@@ -16,6 +16,8 @@ import models from '../../server/models';
 import { PayoutMethodTypes } from '../../server/models/PayoutMethod';
 import { randEmail, randUrl } from '../stores';
 
+export { PAYMENT_METHOD_SERVICES, PAYMENT_METHOD_TYPES, PayoutMethodTypes };
```

Comment:
**[:ice_cream: Details & personal preferences]** Sounds like unnecessary complexity. Why not importing directly from the `/constants` files?

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
The idea is to allow tests to import these constants from the fake-data in order to reduce the number of imports we have in our tests.
Do you think this will cause more confusion?